README.md

```diff
@@ -1,5 +1,57 @@
 
-Climate data analysis, database, and visualization package for Python
+# aospy: automated climate data analysis and management
+## Motivations
+Climate models generally output a wide array of useful quantities, but almost invariably not all needed quantities are directly outputted.  Even for those that are, further slicing and dicing in time and/or space are required.  Moreover, these multiple computations and spatiotemporal reductions are needed for not just a single simulation but across multiple models, simulations, time durations, subsets of the annual cycle, and so on.  Performing these computations across all of the desired parameter combinations quickly becomes impractical without some automation.  But even once some automation is in place, the resulting plethora of data quickly becomes unusable unless it is easily found and imbued with sufficient metadata to describe precisely what it is and how it was computed.
 
-A major improvement in the documentation is currently under way.  In the meantime, please accept our apologies for the lack of information.  And, if you aren't scared away yet, contact Spencer Hill, spencerahill@gmail.com, for more information.
+## What aospy does
+This package provides tools for automating computations involving gridded climate data and the management of the results of those computations.  This enables the user to perform in parallel such commonly needed tasks as:
+
+* repeating a calculation across multiple simulations in a single climate model
+* repeating a calculation across the same simulations performed in multiple models, even if variable names or other quantities differ among the models or simulations
+* repeating a calculation across multiple timespans, both in terms of the start date and end date and in terms of sub-annual sampling: e.g. annual mean, seasonal-means, monthly means.
+* Computing multiple statistical  (e.g. mean, standard deviation) and physical (e.g. column integrals or averages; zonal integrals or averages) reductions on any given computation, both on a gridpoint-by-gridpoint basis and over an arbitrary number of geographical regions
+* any combination of the above, plus many more!
+
+In fact, aospy enables the user to specify an arbitrary combination of models, simulations, variables to compute, time ranges, months of the year, input data types, output data types, and other parameters and to submit them all for execution with a single command.  The desired computations are then executed in parallel, and their outputs are stored as netCDF files in a highly structured directory format with metadata imbued within (1) the netCDF files themselves, (2) the filenames, and (3) the directory structure.  This prevents the all-to-common occurrence of computations becoming that get saved to disk becoming obsolete as time passes (or for other users), due to insufficient metadata describing precisely what they are and how they were computed.
```

Comment:
>The desired computations are then executed in parallel...

This is a bit of a nit, but technically ```aospy``` does not do permutation or parallelization; it is the way our current main.py script is set up in our object libraries that enables this.  Maybe rephrase this as "The desired computations *can then be* executed in parallel..."?